### DISCUSSION TITLE
Pocketbase 0.80 with some breaking changes

### DISCUSSION THREAD

**Discussion by Mashiane** [pockethost/pockethost#116] [Q&A]
Good day

Thanks for this awesome tool.

Are you planning on updating to version 0.80 soon? I understand it has some manual commands to upgrade and also one needs to change their code for "users" management.

I deployed my app with you to test and locally I am running 0.80, which will then make my deployments unusable for user profile management as i am using the javascript sdk on my app.

Thanks

**Comment by benallfree** [comment]
Hi @Mashiane, PocketBase 0.8.0 is already supported. I'm guessing you mean that you have an existing instance that is not running 0.8.0?

<img width="467" alt="image" src="https://user-images.githubusercontent.com/1068356/204854450-318f1abe-8c78-4838-8002-9d60e65f7d99.png">

Changing the version an instance is running is planned in #72 and #60. 

In the mean time, I am happy to manually migrate your instance to 0.8.0. Please email ben@pockethost.io and I'll help you further :)
